### PR TITLE
Add metadata for UiPathDocumentOCR_CPU 22.2

### PR DESCRIPTION
**Pull request by irinel-birsan-uipath**

### CODE REVIEW

File: metadata/UiPathDocumentOCR_CPU__1__metadata.json [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  "outputDescription": "Please refer to the documentation of the Activity used to query the ML Skill.",
+  "processorType": "CPU",
+  "projectId": "[project-id]",
+  "retrainable": false,
+  "stagingUri": "[staging-uri]",
+  "projectName": "UiPath Document Understanding",
+  "projectDescription": "UiPath models to classify and extract information from images and pdfs.",
+  "tenantName": "UiPath",
+  "minAIFabricVersion": "v21.10",
+  "languageVersion": 3,
```
Comment (mohitseth-nitrr) [inline metadata/UiPathDocumentOCR_CPU__1__metadata.json:21]: @irinel-birsan-uipath  i thought we were introducing version 4 for new versions of models. Is that postponed ?

Comment (irinel-birsan-uipath) [reply metadata/UiPathDocumentOCR_CPU__1__metadata.json:21]: @mohitseth-nitrr This is an off-band release for existing SF

**Review by mohitseth-nitrr — APPROVED** [review]
LGTM

File: metadata/UiPathDocumentOCR_CPU__1__metadata.json [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  "version": 1,
```
Comment (vivekjhauipath) [inline metadata/UiPathDocumentOCR_CPU__1__metadata.json:22]: is this version correct?

**Review by vivekjhauipath — APPROVED** [review]
LGTM

File: metadata/UiPathDocumentOCR_CPU__1__metadata.json (hunk 2 above)
Comment (irinel-birsan-uipath) [reply metadata/UiPathDocumentOCR_CPU__1__metadata.json:22]: Yes, it is the first version optimized for CPU

File: metadata/UiPathDocumentOCR_CPU__1__metadata.json [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  "name": "UiPathDocumentOCR_CPU",
+  "outputDescription": "Please refer to the documentation of the Activity used to query the ML Skill.",
+  "processorType": "CPU",
+  "projectId": "[project-id]",
+  "retrainable": false,
+  "stagingUri": "[staging-uri]",
+  "projectName": "UiPath Document Understanding",
+  "projectDescription": "UiPath models to classify and extract information from images and pdfs.",
+  "tenantName": "UiPath",
+  "minAIFabricVersion": "v21.10",
```
Comment (irinel-birsan-uipath) [inline metadata/UiPathDocumentOCR_CPU__1__metadata.json:20]: @vivekjhauipath the next version is `v21.10.1`?

**Pull request merged**